extension/test/newman/collection.json

```diff
@@ -106,9 +284,394 @@
 											"const desiredCartItem = response.cartItems.find(item => item.product && item.product.id === '370')",
 											"pm.test('Existing cart item with input text', () => {",
 											"    const {product: {properties}} = desiredCartItem",
+											"    pm.expect(desiredCartItem).to.be.an('object')",
+											"    pm.expect(properties).to.be.an('array')",
+											"    const textInputProperty = properties[0]",
+											"    pm.expect(textInputProperty.type).to.equal('option')",
+											"    pm.expect(textInputProperty.label).to.equal('Monogramming')",
+											"    pm.expect(textInputProperty.value).to.equal('DEMO Text')",
+											"})",
+											""
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "POST",
+								"header": [
+									{
+										"key": "Content-Type",
+										"value": "application/json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{}"
+								},
+								"url": {
+									"raw": "{{domain}}{{endpoint_cart_get}}",
+									"host": [
+										"{{domain}}{{endpoint_cart_get}}"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "Add a simple product with invalid text field input",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"id": "5cd67dd2-faf6-4720-8031-cbf3a65d0a32",
+										"exec": [
+											"const response = pm.response.json()",
+											"",
+											"pm.test('Status code is 500', function () {",
+											"    pm.response.to.have.status(500);",
+											"});",
+											"",
+											"pm.test('Error is thrown', function () {",
+											"    pm.expect(response.error.code).to.equal('EINVALIDITEM')",
+											"});"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "POST",
+								"header": [
+									{
+										"key": "Content-Type",
+										"value": "application/json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n\t\"products\": [\n\t\t{\n\t\t\t\"productId\": 553,\n\t\t\t\"quantity\": 1,\n\t\t\t\"properties\": [\n\t\t\t\t{\n\t\t\t\t\t\"type\": \"input\",\n\t\t\t\t\t\"id\": \"7\",\n\t\t\t\t\t\"value\": \"DEMO Text is too long\"\n\t\t\t\t}\n\t\t\t\t]\n\t\t}\n\t\t]\n}"
+								},
+								"url": {
+									"raw": "{{domain}}{{endpoint_cart_products_add}}",
+									"host": [
+										"{{domain}}{{endpoint_cart_products_add}}"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "Add a simple product with invalid input id",
+							"event": [
```

Comment:
move under `Errors` folder for better organization else we will end up with sphagetti tests